api/src/utils/apply-query.ts

```diff
@@ -281,6 +283,37 @@ export function applySort(
 			column[0] = column[0]!.substring(1);
 		}
 
+		// Is the column name one of the aggregate functions used in the query if there is any?
+		if (Object.keys(aggregate ?? {}).includes(column[0]!)) {
+			// If so, return the column name without the order prefix
+			const operation = column[0]!;
+
+			// Get the field for the aggregate function
+			const field = column[1]!;
+
+			// If the operation is countAll there is no field.
+			if (operation === 'countAll') {
+				return {
+					order,
+					column: returnRecords ? column[0] : 'countAll',
```

Comment:
```suggestion
					column: returnRecords ? operation : 'countAll',
```
Considering we've already stored `column[0]` in a variable lets use that instead.

This one confuses me. considering we're checking `operation === 'countAll'` and `operation = column[0]` doesnt that meant that this ternary is effectively `returnRecords ? 'countAll' : 'countAll'` 🤔